pkg/fanal/artifact/repo/git_test.go

```diff
@@ -36,6 +38,18 @@ func setupGitRepository(t *testing.T, repo, dir string) (*httptest.Server, *git.
 	return gs, r
 }
 
+// wrapNewArtifact captures stdout to test that `git clone` output is not mixed with formatted output
+func wrapNewArtifact(target string, c cache.ArtifactCache, fs *walker.FS, opts artifact.Option) (func(), []byte, error) {
+	orig := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
```

Comment:
Overriding os.Stdout could cause a problem if this test is run in parallel.

Reply:
agreed -- reverted the test